test/config.yaml

```diff
@@ -1,10 +1,15 @@
 ---
-logVLevel: 4
 nodeName: node1
 nodeIP: '1.2.3.4'
-cluster:
-  url: https://1.2.3.4:6443
-  clusterCIDR: '10.20.30.40/16'
-  serviceCIDR: '40.30.20.10/16'
-  domain: cluster.local
+apiServer:
+  bindHost: https://1.2.3.4:6443
+network:
+  clusterNetwork:
+    - cidr: '10.20.30.40/16'
+  serviceNetwork:
+    - '40.30.20.10/16'
   serviceNodePortRange: 30000-32767
+dns:
+  baseDomain: 'cluster.local'
+debugging:
+  logLevel: 'Normal'
```

Comment:
For the sake of keeping the same value as before and to be consistent with the other tests in https://github.com/openshift/microshift/blob/main/pkg/config/config_test.go, it would be better if we set the logLevel to Debug here.